installation/single_machine_docker.rst

```diff
@@ -7,15 +7,23 @@ This section describes setting up a Citus cluster on a single machine using dock
 
 **1. Install docker and docker-compose**
 
-Follow the installation instructions for `Docker Engine <https://docs.docker.com/engine/installation/>`_ and `Docker Compose <https://docs.docker.com/compose/install/>`_ on your platform.
+* Install the `Docker Platform <https://www.docker.com/products/overview#/install_the_platform>`_.
+* (Linux only) On most platforms the previous step will install docker-compose as well as Docker, but on Linux you'll need to install docker-compose manually by downloading the binary and putting it in your path:
+
+  .. code-block:: bash
+
+    # Linux ONLY
+
+    curl -L https://github.com/docker/compose/releases/download/1.11.1/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
```

Comment:
It's super annoying that we have to ask users to add a repo to their box (to download Engine) only to turn around and ask them to curl some binary into their path.

docker/compose#2235 is a request to add `docker-compose` to the repository. I have _no clue_ why this isn't already the case, but dropping a reference here in order to register my displeasure with upstream.